AK/StringImpl.h

```diff
@@ -33,6 +33,12 @@
 #include <AK/Types.h>
 #include <AK/kmalloc.h>
 
+#ifdef KERNEL
+#    include <Kernel/StdLib.h>
+#else
+#    include <string.h>
+#endif
```

Comment:
Instead of pulling in headers, we can also use `__builtin_memcmp` :]

Reply:
Done!